app/model/editions/templates/feast/FeastSouthernHemisphere.scala

```diff
@@ -0,0 +1,51 @@
+package model.editions.templates.feast
+
+import com.gu.mobile.notifications.client.models.Editions
+import model.editions.templates.TemplateHelpers.{collection, front}
+import model.editions.{CapiDateQueryParam, CapiTimeWindowConfigInDays, Daily, Edition, EditionTemplate, FrontTemplate}
+
+import java.time.ZoneId
+
+object FeastSouthernHemisphere extends FeastEdition {
+  override val title: String = "Feast app [Southern hemisphere]"
+  override val edition = Edition.FeastSouthernHemisphere.entryName
+  override val locale = Some("en_GB")
+  override val notificationUTCOffset = 0
+
+  val MainFront: FrontTemplate = front(
+    "Southern hemisphere",
+    collection("Dish of the day"),
+    collection("Collection 2"),
+    collection("Collection 3"),
+    collection("Collection 4"),
+    collection("Collection 5"),
+    collection("Collection 6"),
+    collection("Collection 7"),
+    collection("Collection 8"),
+    collection("Collection 9")
+  )
+
+  val MeatFreeFront: FrontTemplate = front(
+    "Southern hemisphere",
+    collection("Dish of the day"),
+    collection("Collection 2"),
+    collection("Collection 3"),
+    collection("Collection 4"),
+  )
+
+  val template: EditionTemplate = EditionTemplate(
+    fronts = List(
+      MainFront -> Daily(),
+      MeatFreeFront -> Daily()
+    ),
+    timeWindowConfig = CapiTimeWindowConfigInDays(
```

Comment:
What does this actually do?

Reply:
I think it affects the period we provide the query we backfill data to – elsewhere, it's described as `contentPrefillTimeWindow`.

In attempting to factor this out, it's clear we need to go deep into the skeleton to factor issue generation out from the prefill logic. It touches lots of code and done incorrectly, affects Editions publication – separate PR?

Reply:
Agreed re. seperate PR